tasks/unscramble-words.md

```diff
@@ -0,0 +1,41 @@
+
+# Stimuli
+## The visual components
+From: Warrier V., Bethlehem R.A., Baron-Cohen S. (2017) The “Reading the Mind in the Eyes” Test (RMET). In: Zeigler-Hill V., Shackelford T. (eds) Encyclopedia of Personality and Individual Differences. Springer, Cham. https://doi.org/10.1007/978-3-319-28099-8_549-1
+> Example of stimuli during RME test (not during task itself): 
+> 
+> ![RME Test Question](https://github.com/Watts-Lab/task-mapping/blob/unscramble-words/images/rme%20test.gif)
+
+
+
+
+# Procedure
+## Steps
+> A task flow describing the stages a participant goes through and a formal specification of any conditional or complex relationships therein. 
+> List the steps that make up the task. This could include a training protocol, pre and post surveys, task rounds, etc.  
+> Describe the relationship between the steps. By default they are assumed to be chronological and have no interdependencies, but if they are not chronological, e.g., a choice changes the order of the steps, that should be described in full detail. Similarly, if the steps are interdependent, e.g., performance on one step influences what is seen on another step, this should be described in full detail.
+> 1. Participants were assigned randomly to groups of 4
+> 2. Each of the groups was assigned either face-to-face collaboration or online collaboration. 
+> 3. All participants had personal laptops and worked with their group members in a shared online system to complete the task.
+> 4. Participants in the face-to-face group were seated in a small room with a round table
+> 5. Participants in the online group were serated randomly in a large room and used a text-chat to communicate with group members
+> 6. Groups were presented with a list of 24 randomly scrambled words. Groups had 2 minutes to unscramble as many of the words as possible. Each word only had one correct answer.
+> 7. Individuals completed the Reading in Mind's Eye (RME) Test. The test consists of 36 images of only the eyes of individuals, and participants are asked to choose among four possible mental states to describe the person whose eyes are pictured. The options are all complex mental states (e.g., shame, guilt, curiosity, desire) rather than simple emotions (e.g., happiness, anger).
+> 8. Individuals completed a Five Factor Personality Inventory. This test measures Extraversion, Agreeableness, Conscientiousness, Openness to Experience, and Neuroticism. Sixty items are responded to on a 1 to 5 scale; the mean for each scale was calculated for each participant.
+
+
+## Instructions
+> 1. Participants were instructued to complete the tasks with their group on their computer
+> 2. Participants were instructured to take a series of personality tests.
+
```

Comment:
I think these should be the exact instructions read to the participants, given this, I would probably remove this and reach out to researchers? I didn't see any language in the supporting documents

Reply:
Sounds good, I'll send out the email and we can hold off till we get a reply